apps/designer/app/designer/features/style-panel/shared/style-info.ts

```diff
@@ -23,31 +57,164 @@ export const getStyleSource = (
   return "preset";
 };
 
+const styleProperties = Object.keys(properties) as StyleProperty[];
+const inheritableProperties = new Set<string>();
+for (const [property, value] of Object.entries(properties)) {
+  if (value.inherited) {
+    inheritableProperties.add(property);
+  }
+}
+
+/**
+ * find all breakpoints which may affect current view
+ */
+export const getCascadedBreakpointIds = (
+  breakpoints: Breakpoint[],
+  selectedBreakpointId?: string
+) => {
+  const sortedBreakpoints = breakpoints
+    .slice()
+    .sort((a, b) => a.minWidth - b.minWidth);
```

Comment:
we have a breakpoints.sort utility, its not hard to implement, but over time, the sorting logic might become more complex if we add maxWidth or any other breakpoint params